stripe/src/main/java/com/stripe/android/Stripe.java

```diff
@@ -184,6 +221,37 @@ public Token createBankAccountTokenSynchronous(
     }
 
     /**
+     * Create a {@link Source} using an {@link AsyncTask} on the default {@link Executor} with a
+     * publishable api key that has already been set on this {@link Stripe} instance.
+     *
+     * @param sourceParams the {@link SourceParams} to be used
+     * @param callback a {@link SourceCallback} to receive a result or an error message
+     */
+    public void createSource(@NonNull SourceParams sourceParams, @NonNull SourceCallback callback) {
```

Comment:
The most batteries-included method that can be called. Note: this can't be particularly well-tested because it forces you to use a background thread, which you can't do in a unit test.